bazel/jsoncpp.BUILD

```diff
@@ -0,0 +1,41 @@
+# Copied from Tensorflow:
```

Comment:
Do you plan to contribute this upstream to https://github.com/open-source-parsers/jsoncpp?  This might be a nice addition for them, along with a little documentation for bazel users.

Reply:
Sure, I'll create PR in jsoncpp later.